scripts/test-crds.sh

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env bash
+
+# ------------------------------------------------------------------------------
+# test-crds.sh
+#
+# This script tests that our legacy v1beta1 versions of our CRDs upgrade cleanly
+# to their v1 versions, and that both the legacy and the v1 versions of the CRDs
+# both function with our controllers.
+#
+# NOTE: these tests are sensitive to which Kubernetes version they are run on.
+#       they will only run on Kubernetes clusters between v1.16.x and 1.21.x
+#       (inclusively) as v1.15.x is too old for the new CRDs and v1.22.x is to new
+#       for the old CRDs. At some point in the future (potentially when <v1.22.x is
+#       no longer supported by the chart) we can remove these tests.
+# ------------------------------------------------------------------------------
+
+# ------------------------------------------------------------------------------
+# Configuration
+# ------------------------------------------------------------------------------
+
+RELEASE_NAME="${RELEASE_NAME:-chart-tests-crds}"
+RELEASE_NAMESPACE="${RELEASE_NAMESPACE:-$(uuidgen)}"
+TEST_ENV_NAME="${TEST_ENV_NAME:-kong-charts-tests}"
+
+set -euo pipefail
+
+kubectl cluster-info --context kind-"${TEST_ENV_NAME}" 1>/dev/null
```

Comment:
what is the purpose of this command if we drop the output?

Reply:
It changes the current context, and even though we don't care about the output `pipefail` will cascade any failure to the test.

Reply:
It's a leftbehind, it was just testing that the context exists, but given your below comments we can drop it if we use the context directly.

Reply:
:+1: 4fdf72c8f6880d9704a243dcb2680a4c78c09db5

Reply:
resolved by 4fdf72c8f6880d9704a243dcb2680a4c78c09db5